torchci/scripts/check_alerts.py

```diff
@@ -221,12 +228,25 @@ def generate_no_flaky_tests_issue() -> Any:
     return issue
 
 
-def update_issue(issue: Dict, issue_number: int) -> Dict:
+def delete_old_comments(old_issue: Any) -> None:
+    # delete old comments from github bot
+    comments = old_issue["comments"]["nodes"]
+    for comment in comments:
+        if comment["bodyText"] == UPDATING_ALERT_COMMENT:
+            requests.delete(
+                f"https://api.github.com/repos/{REPO_OWNER}/{TEST_INFRA_REPO_NAME}/issues/comments/{comment['databaseId']}")
+
+
+def update_issue(issue: Dict, old_issue: Any) -> Dict:
     print("Updating issue", issue)
+    delete_old_comments(old_issue)
     r = requests.patch(
-        UPDATE_ISSUE_URL + str(issue_number), json=issue, headers=headers
+        UPDATE_ISSUE_URL + str(old_issue["number"]), json=issue, headers=headers
     )
     r.raise_for_status()
+    r = requests.post(f"https://api.github.com/repos/{REPO_OWNER}/{TEST_INFRA_REPO_NAME}/issues/{old_issue['number']}/comments",
```

Comment:
Does it sync correctly if we just update the comment body here, i.e. including a timestamp?  It would save one additional GitHub API call I think

Reply:
afaik it doesnt sync
removed comment deletion but now the issue might have a lot of spam

Reply:
Noted.  Let's see how this goes